slackbot-api/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python
+WORKDIR /usr/src/app
+COPY ".env" .
+COPY . .
+RUN pip install -r requirements.txt &&\
```

Comment:
why not two RUN commands

Reply:
increases image layers.
idk why thats bad but its not good.
https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-which-is-better

Reply:
ok valid, thx